Markdown/cmm-2.0.14/system-operators/2-installation.md

```diff
@@ -1124,10 +1136,18 @@ $ python3 /usr/share/openstack-dashboard/manage.py collectstatic --noinput
 $ python3 /usr/share/openstack-dashboard/manage.py compress --force
 ```
 
-> **Note:** Expected CSS Error messages:\
-  `ERROR:scss.ast:Function not found: twbs-font-path:1`\
-  `ERROR:scss.compiler:Mixin not found: dropdown-arrow:0`\
-  `ERROR:scss.compiler:Maximum number of supported selectors in Internet Explorer (4095) exceeded!`
+> **Note:** Expected messages: \
+> `WARNING:root:"dashboards" and "default_dashboard" in (local_)settings is DEPRECATED` \
+> `ERROR:scss.ast:Function not found: twbs-font-path:1` \
+> `ERROR:scss.compiler:Mixin not found: dropdown-arrow:0` \
+> `ERROR:scss.compiler:Maximum number of supported selectors in Internet Explorer (4095) exceeded!`
```

Comment:
I think, we can drop the message related to Internet Explorer, since it's not supported any longer

Reply:
It is right that IE is not supported, but better warn in advance about the expected Error messages.